Cargo.toml

```diff
@@ -135,7 +135,7 @@ once_cell = "1.18"
 opentelemetry = { version = "0.24.0" }
 opentelemetry-http = { version = "0.13.0" }
 opentelemetry_sdk = { version = "0.24.0" }
-parking_lot = { version = "0.12" }
+parking_lot = { version = "0.12", features = ["arc_lock", "send_guard"] }
```

Comment:
This is needed for Send-able MutexGuards which allows me to lock snapshot writing in the producer, and unlock it from a future.

Reply:
Got rid of this in favor of AtomicBool; it was unnecessary anyway as I could have used an owned guard from the Tokio mutex, which I missed.